src/JsonSchema/Command/JsonSchemaGenerateCommand.php

```diff
@@ -0,0 +1,98 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Core\JsonSchema\Command;
+
+use ApiPlatform\Core\Api\OperationType;
+use ApiPlatform\Core\JsonSchema\SchemaFactoryInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Exception\InvalidOptionException;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Input\InputOption;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Style\SymfonyStyle;
+
+/**
+ * Generates a resource JSON Schema.
+ *
+ * @author Jacques Lefebvre <jacques@les-tilleuls.coop>
+ */
+final class JsonSchemaGenerateCommand extends Command
+{
+    private $schemaFactory;
+
+    private $formats;
+
+    public function __construct(SchemaFactoryInterface $schemaFactory, array $formats)
+    {
+        $this->schemaFactory = $schemaFactory;
+        $this->formats = $formats;
+
+        parent::__construct();
+    }
+
+    /**
+     * {@inheritdoc}
+     */
+    protected function configure()
+    {
+        $this
+            ->setName('api:json-schema:generate')
+            ->setDescription('Generates the JSON Schema for a resource operation.')
+            ->addArgument('resource', InputArgument::REQUIRED, 'The Fully Qualified Class Name (FQCN) of the resource')
+            ->addOption('itemOperation', null, InputOption::VALUE_OPTIONAL, 'The item operation')
+            ->addOption('collectionOperation', null, InputOption::VALUE_OPTIONAL, 'The collection operation')
+            ->addOption('format', null, InputOption::VALUE_OPTIONAL, 'The response format', array_key_first($this->formats))
+            ->addOption('output', null, InputOption::VALUE_NONE, 'Use this option to get the output version');
```

Comment:
in every other command provided, `output` is used to specify a file path to write to a file.
Can we rename this one ?

Reply:
Do you have a suggestion? `request`/`response`?

Reply:
Just use `--type=input|output`?